tests/functional/tests/mon/test_mons.py

```diff
@@ -0,0 +1,29 @@
+
+class TestMons(object):
+
+    def test_ceph_mon_package_is_installed(self, CephNode, Package):
+        assert Package("ceph-mon").is_installed
+
+    def test_mon_listens_on_6789(self, CephNode, Socket):
+        assert Socket("tcp://%s:6789" % CephNode["address"]).is_listening
+
+    def test_mon_service_is_running(self, CephNode, Service):
+        service_name = "ceph-mon@ceph-%s" % CephNode["vars"]["inventory_hostname"]
+        assert Service(service_name).is_running
+
+    def test_mon_service_is_enabled(self, CephNode, Service):
+        service_name = "ceph-mon@ceph-%s" % CephNode["vars"]["inventory_hostname"]
+        assert Service(service_name).is_enabled
+
+    def test_can_get_cluster_health(self, CephNode, Command):
+        output = Command.check_output("sudo ceph -s")
```

Comment:
Alright, I don't mind keeping the `ceph -s` but we really need support for cluster name here. I pushed https://github.com/ceph/ceph-ansible/pull/1153 this morning, we can discuss if it's the right approach.